miners/text/template_miner.py

```diff
@@ -371,16 +371,14 @@ def run( self ):
 
                     # ---- Run epoch ----
                     total_epoch_loss = 0.0
-                    start_block = self.neuron.subtensor.get_current_block() + 1
+                    start_block = self.neuron.subtensor.get_current_block()
                     end_block = start_block + self.config.miner.epoch_length
                     block_steps = [ start_block + block_delta for block_delta in range(start_block, end_block)]
-                    print(block_steps)
                     progress_bar = qqdm( block_steps, total=len(block_steps), desc=format_str('white', f'Epoch:'))
                     for block in progress_bar:
-
                         # --- Iterate over batches until the end of the block.
                         current_block = self.neuron.subtensor.get_current_block()
-                        while block <= current_block:
+                        while block >= current_block:
```

Comment:
This will cause a run away issue where the blocks just progress quickly while the current block is slow.